{
  "name": "@stdlib/utils-async-try-then",
  "version": "0.1.0",
  "description": "If a function does not return an error, invoke a callback with the function result; otherwise, invoke a second function.",
  "license": "Apache-2.0",
  "main": "./index.js",
  "homepage": "https://stdlib.io",
  "repository": {
    "type": "git",
    "url": "git://github.com/stdlib-js/utils-async-try-then.git"
  },
  "bugs": {
    "url": "https://github.com/stdlib-js/stdlib/issues"
  },
  "keywords": [
    "stdlib",
    "stdutils",
    "stdutil",
    "utilities",
    "utility",
    "utils",
    "util",
    "asynchronous",
    "async",
    "try",
    "catch",
    "then",
    "try-catch",
    "try-then",
    "trap",
    "control",
    "flow",
    "condition",
    "conditional",
    "either-or"
  ],
  "funding": {
    "type": "opencollective",
    "url": "https://opencollective.com/stdlib"
  }
}
